lib/ansible/modules/system/firewalld.py

```diff
@@ -181,6 +201,39 @@
     permanent: yes
     immediate: yes
     state: enabled
+  loop:
+    - ipv4
+    - ipv6
+
+- firewalld:
+    chain: sshg
+    fw_family: ipv4
+    table: filter
+    permanent: true
+    state: enabled
+
+- firewalld:
+    chain: sshg
+    fw_family: ipv4
+    table: filter
+    direct_rule: '-m tcp -p tcp --dport 332 -j ACCEPT'
```

Comment:
No need to quote this line.
```suggestion
    direct_rule: -m tcp -p tcp --dport 332 -j ACCEPT
```